tests/actors.rs

```diff
@@ -0,0 +1,125 @@
+use crate::common::spawn_app;
+use sqlx::{Pool, Postgres};
+use std::collections::HashMap;
+
+mod common;
+
+// We use sqlx::test so sqlx will create a database instance for the test,
+// set it up based on the migration data, and shut it down upon success.
+// The pool passed to the function is that created database.
+#[sqlx::test]
+async fn get_actors_works(pool: Pool<Postgres>) {
+    let app = spawn_app(&pool, "actors").await;
+    // Insert a value into the empty db so we have something to get
+    sqlx::query!("INSERT INTO actors (name) VALUES ($1)", "TestName")
+        .execute(&pool)
+        .await
+        .unwrap();
+
+    let response = app.client.get(app.url).send().await.unwrap();
+
+    assert_eq!(response.status().as_u16(), 200);
+    let expected = r#"[{"id":1,"name":"TestName"}]"#;
+    let content = response.text().await.unwrap();
```

Comment:
Why is text() being awaited? Isn't there a way to get the response string without parsing or other work being done here?